utilities/casebundle-cronjob/cronjob.yaml

```
@@ -0,0 +1,32 @@
+                    key: egov-state-level-tenant-id 
+              - name: USER_SERVICE_HOST
+                valueFrom:
+                  configMapKeyRef:
+                    name: egov-service-host
+                    key: egov-user
+              - name: CASEBUNDLE_HOST
+                valueFrom:
+                  configMapKeyRef:
+                    name: egov-service-host
+                    key: casemanagement      
+          restartPolicy: OnFailure
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)